amazon-ml-hyperparameter-optimization/README.md

```diff
@@ -58,7 +58,7 @@ The [hyperparameters](http://docs.aws.amazon.com/machine-learning/latest/dg/trai
 
 Regularization roughly quantifies the idea in machine learning that simpler explanations are better. This [Quora answer](https://www.quora.com/What-is-regularization-in-machine-learning) has a great explanation of what regularization is and why it matters.
 
-Hyperparameter optimization works to maximize some performance metric of your model. This example maximizes the Area Under the (Receiver Operating Characteristic) Curve (AUC) of a binary classifier because it is easily exposed by Amazon Machine Learning. Alternatives include maximizing accuracy or minimizing error (maximizing negative error). Read more about how Amazon measures model performance on [Binary Model Insights](http://docs.aws.amazon.com/machine-learning/latest/dg/binary-model-insights.html).
+Hyperparameter optimization works to maximize some performance metric of your model. This example maximizes the Area Under the (Receiver Operating Characteristic) Curve (AUC) of a binary classifier because it is easily exposed by Amazon Machine Learning. Alternatives include maximizing accuracy or minimizing error. Read more about how Amazon measures model performance on [Binary Model Insights](http://docs.aws.amazon.com/machine-learning/latest/dg/binary-model-insights.html).
```

Comment:
"Hyperparameter optimization works to `optimize` some performance metric of your model."?